src/services/supabase.ts

```diff
@@ -42,12 +42,7 @@ export enum RPCS {
 
 export const supabaseClient = createClient(
     supabaseSettings.url,
-    supabaseSettings.anonKey,
-    {
-        // https://github.com/estuary/ui/issues/87
-        // This is not working correctly so disabling for now.
-        autoRefreshToken: false,
-    }
```

Comment:
Even though this does not always work it is WAY worse to keep getting logged out every hour.